index.html

```diff
@@ -0,0 +1,186 @@
+		</div>
+		<div class="recent_work_container">
+			<img src="./images/imgs.png" alt="works_section">
+			<div class="recent_work_container_text">
+				<h3>Multi-Post Stories</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div class="butons_container">
+				<button type="button">CSS</button>
+				<button type="button">HTML</button>
+				<button type="button">Boostrap</button>
+				<button type="button">Ruby</button>
+			</div>
+			<button  type="button" class="project_btn">See project</button>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<div class="box">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button  type="button"class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button" class="project_btn btn">See project</button>
+			</div>
+			
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
+				</div>
+				<button type="button"  class="project_btn btn">See project</button>
+			</div>
+			
+			<div class="box ">
+				<h4>Profesional Art Printing Data</h4>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+				<div class="butons_container grid">
+					<button type="button">HTML</button>
+					<button type="button">Boostrap</button>
+					<button type="button">Ruby</button>
```

Comment:
According to the project requirements, kindly use `ul` and `li` for categories in the "Work" section. `Html`, `Bootstrap`, and `Ruby` should be list items.